backend/README.md

```diff
@@ -311,92 +305,59 @@ Run with (supervisord.conf should be provided, see [supervisord.conf default_pat
     supervisord
 
 
-Running Pushpin Service
----------------
+## Running Pushpin Service
 
 For a working Tracim instance, you need to setup pushpin as proxy for tracim web service.
 
 See [main readme](../README.md)  section _Install and run pushpin for UI updates_
 
-## Run Tests and Others Checks ##
+## Lint the code
 
-### Run Tests ###
-
-Some functional tests need additional daemons that are run through docker:
-
-```shell
-sudo apt install docker.io docker-compose
-```
-
-Some directories are required to make tests functional, you can create them and do some other check
-with this script:
-
-    # in the backend folder
-    python3 ./setup_dev_env.py
-
-Before running some functional test related to email, you need a local working *MailHog*
-see here: https://github.com/mailhog/MailHog
-
-You can run it this way with Docker:
-
-    docker pull mailhog/mailhog
-    docker run -d -p 1025:1025 -p 8025:8025 mailhog/mailhog
-
-You need also a test ldap server on port 3890 for ldap related test.
-See here: https://github.com/rroemhild/docker-test-openldap
-
-You can run it this way with Docker:
-
-    docker pull rroemhild/test-openldap
-    docker run -d -p 3890:389 rroemhild/test-openldap
-
-You need also a elasticsearch server on port 9200 for elasticsearch related test
-You can run it this way with Docker Compose:
+Install the required versions:
 
-    # from backend folder
-    docker-compose up elasticsearch
+    pip install -r requirements-devtool.txt
 
-You also need a minio server on port 9000 for tests related to s3.
-You can run it this way with Docker Compose:
+Then run flake8:
 
-    # from backend folder
-    docker-compose up minio
+    flake8
 
-please note that syntax is similar to minio command line, "minio server /data" will start the minio server and use
-/data dir (in our case, it's the /data of the docker and his content will be deleted after stopped the docker as we do not
-create a docker volume), see minio documentation for more info: https://docs.min.io/docs/minio-docker-quickstart-guide.html .
+## Run Tests
 
-Create default test_storage_dir folder tree to make test work properly out of the box:
+First setup the needed tools and directories (only needed once):
 
+    # from backend directory
+    python3 ./setup_dev_env.py
     ./create_test_storage_dir.sh
+    # several external services (mail/databases/…) are started through a docker compose file
+    sudo apt install docker.io docker-compose
+    # add the current user to docker group, you'll need to use a new login shell for this change to be taken into account
+    sudo usermod -a -G docker $USER
```

Comment:
I think it's better to just add a link to the docker official documentation, but if you are sure this command works on every OS where we can run tracim, that is ok

- https://docs.docker.com/compose/install/
- https://docs.docker.com/engine/install/debian/

Reply:
Most of the file directly lists installation instructions for Debian systems, I added the links to the official installation instructions + a comment to indicate that those 2 lines are valid for Debian.